### ISSUE TITLE
Compliance/Compliance dashboard - Links to My Compliance tabs

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-compliance#58]
#### Description:

Clicking on one of the the links in the 'My Compliance' tile on the COAM dashboard will open one of two tabs.

[Figma design](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20107-1497&t=wbgmWfTWJPzifM0P-1&scaling=scale-down&content-scaling=fixed&page-id=14307%3A1663&starting-point-node-id=20107%3A1497&show-proto-sidebar=1)

#### Acceptance Criteria:

As an operator, when I click on one of the three links in the My Obligation tile, each link opens to a separate tab:

- [ ] Compliance summaries link opens the [Compliance summary tab][(url)](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20107-1332&t=mMCqKm8zClEjGn5g-4)
- [ ] Payment summary link opens the [Payment Summary tab](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=24063-2&t=mMCqKm8zClEjGn5g-4)

Post-MVP consideration:
- [ ] Imposed penalty link opens the [Imposed Penalties tab ](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=25048-16&t=mMCqKm8zClEjGn5g-4)

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #57 
Blocking #17 

#### Notes:

-

**Comment by fviduya** [comment]
I have tested this in the DEV environment, [here is the test document for reference](https://buttoninc-my.sharepoint.com/:w:/g/personal/francesca_button_is/EajXUZgeEAdEkFH-JO_G1s4BKE-4sRnoqA1yF04avIpsuQ?e=QJXj0y).

I can verify that:
- [x] Clicking the Compliance tile navigates to the Compliance dashboard
- [x] Clicking ```Compliance Summaries``` under the My Compliance tile navigates to the Compliance Summaries tab and breadcrumbs show accordingly
- [x] Clicking ```Payment Summaries``` under the My Compliance tile navigates to the Payment Summaries tab and breadcrumbs show accordingly 

Moving to Pending PO. Thank you!

**Comment by vesselak** [comment]
Reviewed QA notes and tested in dev. Accepting ticket.
